src/OrchardCore.Modules/OrchardCore.Users/Manifest.cs

```diff
@@ -56,6 +56,15 @@
     Category = "Settings"
 )]
 
+[assembly: Feature(
+    Id = "OrchardCore.Users.Localization",
+    Name = "User Localization",
+    Description = "Provides a way to set the culture per user.",
+    Dependencies = new[] { "OrchardCore.Users", "OrchardCore.Localization" },
+    Category = "Settings",
+    Priority = "-1"
```

Comment:
Comment why this is necessary.

Reply:
@hishamco added this

Reply:
This has been added to avoid changing the order in the localization module